server/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     // "paths": {},                                      /* Specify a set of entries that re-map imports to additional lookup locations. */
     // "rootDirs": [],                                   /* Allow multiple folders to be treated as one when resolving modules. */
     // "typeRoots": [],                                  /* Specify multiple folders that act like './node_modules/@types'. */
-    // "types": [],                                      /* Specify type package names to be included without being referenced in a source file. */
+    "types": ["node", "mocha"] /* Specify type package names to be included without being referenced in a source file. */,
```

Comment:
mocha vs jest? :)

Reply:
either it's fine, for now I added mocha since we uses this one